charts/kubeclarity/templates/deployment.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ include "kubeclarity.name" . }}
+{{- if .Values.kubeclarity.affinity }}
+      affinity:
+{{ toYaml .Values.kubeclarity.affinity | indent 8 }}
+{{- end }}
```

Comment:
One minor styling thing here. To avoid weird new lines when the template is rendered its better to use nindent instead of indent. Its also nice to align the if statements with the indent of the part of the template that is being added. For example:

```
  spec:
      serviceAccountName: {{ include "kubeclarity.name" . }}
      {{- if .Values.kubeclarity.affinity }}
      affinity: {{- toYaml .Values.kubeclarity.affinity | nindent 8 }}
      {{- end }}
```

Reply:
Doesn't seem to make any difference if you chomp the leading whitespace with `{{-` and use `nindent` vs  `{{` and `indent` when I tested, but have changed to keep consistent with the rest of the chart.

Likewise most of the other conditionals in the template are not aligned with the actual data, so I was just trying to keep consistent there too. Have changed it though